config/crd/bases/infrastructure.cluster.x-k8s.io_gcpclusters.yaml

```diff
@@ -34,7 +34,6 @@ spec:
     listKind: GCPClusterList
     plural: gcpclusters
     singular: gcpcluster
-  preserveUnknownFields: false
```

Comment:
I don't think we actually want to remove this field.

Reply:
https://github.com/kubernetes-sigs/cluster-api/blob/546d56bea2b8cd9ac24d90ed7599d209dfe2ccc5/Makefile#L187